resources/default.rb

```diff
@@ -140,6 +142,38 @@
     end
   end
 
+  nexus3_license 'update installed license' do
+    action :nothing
```

Comment:
Shouldn't we use `:update` as default action with a guard if the file doesn't exist?

We could also add a global condition in the resource to define them only when there is a license expected. Not all the users of nexus will have a license

Reply:
The `license` property allows `nil` and the `nexus3_license` is (should be) idempotent so I guess we should use `:update` indeed but without condition

Reply:
Nexus need to already have started, that's the reason of the notification.

Reply:
As discussed, if we directly do update we need to handle nexus being down in the api lib which is not that great